python-package/xgboost/compat.py

```diff
@@ -43,6 +43,11 @@ def lazy_isinstance(instance: Any, module: str, name: str) -> bool:
     pandas_concat = None
     PANDAS_INSTALLED = False
 
+
+# cuDF
+CUDF_INSTALLED = importlib.util.find_spec("cudf") is not None
```

Comment:
`importlib` is not a python standard library.

Reply:
It is part of CPython? https://docs.python.org/3/library/importlib.html

Reply:
Interesting, thank you for the reference.